### PR TITLE
update examples for wd.js client

### PR DESCRIPTION
**Pull request by dmlemeshko**
## Proposed changes

Update `wd.js` examples based on latest client [release](https://github.com/admc/wd/releases)

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by dpgraham** [comment]
Thanks for the contribution @dmlemeshko 

These `.md` files actually are auto-generated and aren't supposed to be edited.... 

There's actually `.yml` files that are supposed to be edited. 

If you look at the top of each of these `.md` files, it will give you a link to the `.yml` file to edit

e.g.)
```[//]: # (DO NOT EDIT THIS FILE! This is an auto-generated file. Editing for this document happens in /commands-yml/commands/device/keys/is-keyboard-shown.yml)```

File: docs/en/commands/session/geolocation/set-geolocation.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ driver.location({latitude: 121.21, longitude: 11.56, altitude: 94.23});
 
 
 // wd example
-let location = await driver.sessionCapabilities('c8db88a0-47a6-47a1-802d-164d746c06aa');
+await driver.setGeoLocation(121.21, 11.56, 10);
```
Comment (dpgraham) [inline docs/en/commands/session/geolocation/set-geolocation.md]: Nice catch!

**Comment by dpgraham** [comment]
And also, I should add, after you edit the YML files you should run `npm run generate-docs` from the root and commit the auto-generated `.md` files.

**Comment by dmlemeshko** [comment]
@dpgraham Thank you for details, I generated docs the proper way. Still I don't understand why there is a message stating `Dmitry Lemeshko seems not to be a GitHub user`.
Commit looks to be valid for me:

```commit f2164ec4509896c7921c1d34b9c74e41407ee27e (HEAD -> update-wd-js-docs, origin/update-wd-js-docs)
Author: Dmitry Lemeshko <dima.inspired@gmail.com>
Date:   Fri Jun 8 11:03:43 2018 +0200
update examples for wd.js client
```
And my account has this email as default.

**Comment by dpgraham** [comment]
I think there may be an issue with the bot.

@jlipps Do you have any ideas on what the problem is?

**Comment by dpgraham** [comment]
Alright, I can merge this now.

@dmlemeshko How were you able to get the CLA working? Did you have to do something or did the bot just take a long time to recognize that you signed it?

**Comment by dmlemeshko** [comment]
@dpgraham I did nothing :) The other day I reloaded this page and saw CLA been passed, so probably there is some delay on bot side

**Pull request merged**